client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -56,20 +52,21 @@ function ReaderSubscriptionListItem( {
 				/>
 			</div>
 			<div className="reader-subscription-list-item__byline">
-				<span className="reader-subscription-list-item__site-title">{ <a href={ streamUrl }> { siteTitle } </a> }</span>
+				<span className="reader-subscription-list-item__site-title">
+					{ <a href={ streamUrl } className="reader-subscription-list-item__link"> { siteTitle } </a> }
+				</span>
+				<div className="reader-subscription-list-item__site-excerpt">{ siteExcerpt }</div>
 				{ ! isEmpty( authorName ) &&
-					<span>
-						<span className="reader-subscription-list-item__by-text">
-							{ translate( 'by' ) }
-						</span>
-						<span><a href={ streamUrl }> { authorName } </a></span>
+					<span className="reader-subscription-list-item__by-text">
+						{
+							translate( 'by {{author/}}', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **26** times:
`translate( 'by %(author)s' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/reader/current-subs)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).